src/BinaryChart.js

```diff
@@ -55,7 +57,8 @@ export default class BinaryChart extends Component {
     shouldComponentUpdate(nextProps) {
         if (
             this.props.symbol !== nextProps.symbol ||
-                this.props.type !== nextProps.type                  // this might break when we have more chart type
+                this.props.type !== nextProps.type ||                 // this might break when we have more chart type
+                this.props.pipSize !== nextProps.pipSize
```

Comment:
pipSize comparison is probably unnecessary, since it already is covered in symbol comparison. pipSize can not change independently of symbol.